backend/src/main/java/ai/verta/modeldb/name_generator/NamesGenerator.java

```diff
@@ -0,0 +1,1179 @@
+package ai.verta.modeldb.name_generator;
+
+import java.util.Random;
+
+/**
+ * MobyNames(https://github.com/moby/moby/tree/master) is a Java port of the Moby Project random
+ * name generator Java: (https://github.com/googlielmo/mobynames)
```

Comment:
@conradoverta This is the origin for the source.  It's not published to maven as a library.  We would need to build the project and use it as a jar.